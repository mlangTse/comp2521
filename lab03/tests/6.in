20
10
15
5
9
3
17
16
19
