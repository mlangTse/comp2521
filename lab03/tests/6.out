Tree:
      20
      /
     10
    / \
   /   \
  5    15
 / \     \
3   9    17
         / \
        /   \
       16   19

#nodes = 9,  #leaves = 4
Infix  : 3 5 9 10 15 16 17 19 20 
Prefix : 20 10 5 3 9 15 17 16 19 
Postfix: 3 9 5 16 19 17 15 10 20 
ByLevel: 20 10 5 15 3 9 17 16 19 
